.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on: [push]
```

Comment:
This is not running on pull request coming from a fork, I think it needs to be changed to run on pull requests as well. To avoid double status when creating a PR, I suggest running on push to master only:

```
on:
  push:
    branches:
      - master
  pull_request:
```